common/utils/iiif.js

```diff
@@ -8,8 +8,7 @@ import {
 } from '../model/iiif';
 
 export function getServiceId(currentCanvas: ?IIIFCanvas) {
-  if (!currentCanvas) return null;
-  const serviceSrc = currentCanvas.images[0].resource.service;
+  const serviceSrc = currentCanvas?.images[0]?.resource?.service;
```

Comment:
Oh nice – I missed this going in.

Reply:
@gestchild you could get it in for TS too, or wait till we use it there.